app.yaml

```diff
@@ -1,5 +1,5 @@
 application: cr-status
-version: 2016-10-27
+version: 2016-08-26-newmodel-ssr2
```

Comment:
I know this is just an arbitrary identifier, but do you want to update it to a more accurate date?